src/soc/intel/common/block/cse/cse.c

```diff
@@ -1095,6 +1095,12 @@ unsigned int cse_get_me_disable_mode(void)
 	if (smmstore_lookup_region(&rdev))
 		return 0;
 
+    /* Disable ME if in Firmware Update Mode */
+	size = sizeof(var);
+	ret = efi_fv_get_option(&rdev, &dasharo_system_features_guid, "FirmwareUpdateMode", &var, &size);
```

Comment:
Like I mentioned in other channel, it may be better to simply invoke `cse_hmrfpo_enable` on FUM path as it will ensure the descriptor is also unlocked, just in case. It will also cover cases where ME region is RO in the descriptor.

In such case we would also need to add `cse_hmrfpo_enable` implementation.

Reply:
The ME_WE pin on the EC may be used for the same purpose, in a later iteration perhaps.

Reply:
https://github.com/Dasharo/dasharo-issues/issues/481